microsoft_student_partners/hackcovid19/organizers.json

```diff
@@ -24,6 +24,15 @@
     "instagram": "https://www.instagram.com/aashrut_vaghani",
     "position": "Tech Lead"
   },
+  "Damilola Omiyera": {
+    "name": "Damilola Omiyera",
+    "image_url": "https://avatars2.githubusercontent.com/u/25831498?s=400&u=b7659fe2973ac4f2ef0600d4dacd9c816816d068&v=4",
+    "linkedin": "https://www.linkedin.com/in/damilola-omiyera-115980150/",
+    "github": "https://github.com/oomis",
+    "twitter": "https://twitter.com/Iam_omis",
+    "instagram": "https://www.instagram.com/r_omis",
```

Comment:
Please use any three social links only it doesn't look good with more than three social links